tests/e2e/performance/test_small_file_workload.py

```diff
@@ -282,9 +329,9 @@ def aggregate_samples_results(self):
                         mean = np.mean(results[self.managed_keys[key]["name"]])
 
                         pct_dev = (st_deviation / mean) * 100
-                        if pct_dev > 5:
+                        if pct_dev > 20:
```

Comment:
Any reason to change the deviation to 20% from 5%?

Reply:
For now, until we will have fix environment for BM & VMware so the results will be more persistent.